test/unit/yesno.spec.ts

```diff
@@ -615,5 +609,45 @@ describe('Yesno', () => {
         expect(yesno.intercepted()).to.have.lengthOf(3);
       });
     });
+
+    describe('#ignore', () => {
+      it('allows ignoring a mocked url', async () => {
+        yesno.mock([createMock(), createMock(), createMock()]);
```

Comment:
Can you make each mock unique so we can verify exactly which mocks are being matched to at each step. That will show b/c the assertions will vary.

Also I think the test is clearer if you create the three mocks up front and move the `.ignore` call to line 616. Then the mocks are all setup and ready to go and not intermingled with running the test and assertions.

Reply:
I can't move the `.ignore` up, because it will then match all of the mocks. The test is showing that the first call returns the mock, then after the ignore is defined, the mock is not returned. Then finally even if a mock is defined with `.respond` the ignore still takes precedence.

Reply:
> Then finally even if a mock is defined with .respond the ignore still takes precedence.

I think you should create a second test for this behavior. Generally you want to test one thing in each test.

Reply:
I'd like to prove in the tests that if the same ignored url is called multiple times it will continue to be mocked. I do think that's appropriate to keep in this test, and then split out the test showing the behavior mixed with `respond` like I mentioned above.